examples/pipelines/commoncrawl/components/download_commoncrawl_segments/fondant_component.yaml

```diff
@@ -0,0 +1,35 @@
+name: Download commoncrawl segment files
+description: Component that downloads commoncrawl segment files based on WARC paths
+image: ghcr.io/ml6team/download_commoncrawl_segments:latest
+
+consumes:
+  segment:
+    fields:
+      path:
+        type: string
+
+produces:
+  webpage:
+    fields:
+      url:
+        type: string
+      html:
+        type: string
+
+args:
+  use_s3:
+    description: Whether to use S3 to download the commoncrawl segment file. Set to True if you are running this component on an AWS cluster.
+    type: bool
+    default: 'False'
```

Comment:
```suggestion
    default: false
```

Aren't you passing a string here instead of a boolean? Same for the argument below.

Reply:
[component_spec.json](https://github.com/ml6team/fondant/blob/0abc5973f82c13b3deca544cd80f9b697e55a14a/src/fondant/schemas/component_spec.json#L84) currently allows only strings and integers for default values

Reply:
Let's update the schema then :)